examples/stubbing__resources/README.md

```diff
@@ -0,0 +1,7 @@
+# Stubbing Ressources
+
+The spec file [cypress/integration/spec.js](cypress/integration/spec.js) shows how to stub ressources that are loaded e.g. by `img` tag.
+
+When clicking onto the button in the `index.html`, an api call is simulated that would provide an image url. This image is manipulated in order to display a predefined image.
+
+![Test example](images/ressources-example.png)
```

Comment:
The ressources-example.png will need to be renamed to resources-example.png also

```suggestion
![Test example](images/resources-example.png)
```